Parking/OnStreetParking/doc/spec.md

```diff
@@ -97,7 +97,20 @@ A `null` or empty value indicates an indefinite duration.
 + `description` : Description about the onstreet parking zone. 
     + Normative References: [https://schema.org/description](https://schema.org/description)
     + Optional
-    
+
++ `status` : Status of the parking site.
```

Comment:
Allowed values for status of an onstreetparking essentially are essentially different than of an offstreetparking. 

To my mind allowed values should be:

`noParkingAllowed`, to represent a temporal closing. We would need a new attribute to represent the closing period. 
`specialParkingRestrictionsInForce`, represents that special restrictions are in force. We would need extra machinery to represent what are restrictions and to which period they apply.
`normalConditions`, the on street parking is under the regular conditions. 

Having said that I think we need as well to revisit the `status`property of `OffstreetParking` and `ParkingSpot`. And see if we need a status property over the `ParkingGroup` entity, which I think we probably need as well.